docker-compose.yml

```
@@ -0,0 +1,9 @@
+version: '3.8'
+services:
+  mongo:
+    image: mongo:7.0
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: mongoadmit
+      MONGO_INITDB_ROOT_PASSWORD: root
+    ports:
+      - 27017:27017
```

```suggestion
      - 27017:27017

```

Please, add new line. Also, it's the good idea to setup your IDE to make it automatically
See https://stackoverflow.com/a/16761228

---

Fixed: Added new lines for all files, and updated IDE settings